Assets/Scripts/Scenes/InGame/Player/TempPlayerMove.cs

```diff
@@ -213,6 +220,28 @@ private IEnumerator CountTiredPenalty()
             yield return new WaitUntil(() => _myPlayerStatus.nowStaminaValue > 10);//�X�^�~�i��10�܂ŉ񕜂���̂�҂�
             _isTiredPenalty = false;
         }
+
+        private IEnumerator CheckParalyze()
+        { 
+            while (true) 
+            {
+                yield return new WaitForSeconds(1.0f);

```

Comment:
```suggestion
yield return new WaitForSeconds(5.0f);
``` 
1秒おきはまずいと思ったので、5秒くらいにしてほしいです。